chapter05/qli/ex2.pl

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+
+chomp( my @inputs = <STDIN> );
+
+print "1234567890"x5, "\n" ;
```

Comment:
Took me a while too but figured we could just do <code>((1..9),0)</code> :sweat_smile: